cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -158,6 +160,10 @@ func (command *Evaluate) Initialize(args []string) (evaluationContext *evaluate.
 		}
 		evaluationContext.Runs = command.Runs
 
+		if command.RuntimeImage == "" {
+			command.RuntimeImage = "ghcr.io/symflower/eval-dev-quality:v" + evaluate.Version
+		}
```

Comment:
awesome! Maybe some tests as well? Just using a plain alpine image and ensure it fails?